cluster/ci/config/prow/plugins.yaml

```diff
@@ -185,6 +185,10 @@ plugins:
   - wip
   - yuks
 
+  openshift/service-catalog:
+  - trigger
+  - docs-no-retest
```

Comment:
This plugin is not used anymore, please remove. I'll make sure we remove it from elsewhere.